test/public/test/call-remote-callbacks.js

```diff
@@ -275,6 +275,29 @@ asyncTest('required radio should only require one to be checked', 1, function()
   }, 13);
 });
 
+asyncTest('required radio should only require one to be checked if not all radios are required', 1, function() {
+  $(document).bind('iframe:loading', function() {
```

Comment:
What version of jQuery is this project using? `.bind()` was deprecated a while back